config/default.js

```diff
@@ -30,6 +30,13 @@ module.exports = {
   // Disables the server side render, handy for debugging.
   disableSSR: false,
 
+  // For all Firefox 57+ (Quantum) UAs, send the `appversion` in all API
+  // search requests. This will return only compatible add-ons, which is
+  // good for UX (it prevents a lot of incompatible old add-ons).
+  // Disable this in development when working with stage data, which is
+  // very out-of-date and mostly not 57+ compatible.
+  checkForVersionCompatiblityInAPIRequests: true,
```

Comment:
This seems to only apply to search so I think it would help to indicate that by name. I would suggest:

```js
restrictSearchResultsToAppVersion: true
```